openstack/compute/v2/extensions/bootfromvolume/requests.go

```diff
@@ -5,26 +5,29 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
 )
 
-// SourceType represents the type of medium being used to create the volume.
-type SourceType string
+// SourceDestinationType represents the type of medium being used as either
+// the source or destination type of the bootable device
+type SourceDestinationType string
 
 const (
-	// Volume SourceType
-	Volume SourceType = "volume"
+	// Volume SourceType and DestinationType
+	Volume SourceDestinationType = "volume"
 	// Snapshot SourceType
-	Snapshot SourceType = "snapshot"
+	Snapshot SourceDestinationType = "snapshot"
 	// Image SourceType
-	Image SourceType = "image"
+	Image SourceDestinationType = "image"
 	// Blank SourceType
-	Blank SourceType = "blank"
+	Blank SourceDestinationType = "blank"
+	// Local DestinationType
+	Local SourceDestinationType = "local"
```

Comment:
How about we leave `SourceType` as-is, prefix the current constants with 'Source', and define a new `DestinationType`:
```go
type (
  DestinationType string
  SourceType string
)

const (
  SourceVolume SourceType = "volume"
  DestinationVolume DestinationType = "volume"
  ...
)
```